clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/DefaultClusteringService.java

```diff
@@ -316,8 +317,13 @@ public void destroy(String name) throws CachePersistenceException {
         try {
           this.entity.validateCache(cacheId, clientStoreConfiguration);
         } catch (ClusteredTierValidationException ex) {
-          if (ex.getCause() instanceof InvalidStoreException) {
-            this.entity.createCache(cacheId, clientStoreConfiguration);
+          if(ex.getCause() instanceof ClusteredClientException) {
```

Comment:
The wrapping / unwrapping that leads to these `instanceof` checks are a design smell IMO